data-service/src/main/java/uk/gov/gchq/palisade/service/data/service/SimpleDataService.java

```diff
@@ -127,8 +129,15 @@ DataReaderRequest constructReaderRequest(final ReadRequest request) {
 
                 LOGGER.debug("Output stream closed, {} processed and {} returned, auditing success with audit service", recordsProcessed.get(), recordsReturned.get());
                 auditRequestComplete(readerRequest, recordsProcessed, recordsReturned);
-            } catch (IOException | NoCapacityException ex) {
+            } catch (IOException | RuntimeException ex) {
                 auditRequestReceivedException(dataRequest, ex);
+                throw new ReadException(ex);
+            } catch (Error err) {
```

Comment:
SQ doesn't like the catching of an Error. This should either be changed to an Exception or ignore the rule within SQ.

Reply:
fixed within SQ